include/rocksdb/db.h

```diff
@@ -1587,6 +1587,15 @@ class DB {
     return Status::NotSupported("EndTrace() is not implemented.");
   }
 
+  virtual Status StartIOTrace(Env* /*env*/, const TraceOptions& /*options*/,
```

Comment:
You need to document the expected behavior of this methods and their arguments.

Reply:
I was thinking that once all changes are done (File System side),  I will add it to History.md, wiki and other required places.

Reply:
Maybe just add a comment saying they are experimental, for now.